docs/sources/alerting/set-up/configure-alertmanager/index.md

```diff
@@ -34,17 +34,32 @@ Grafana Alerting is based on the architecture of the Prometheus alerting system.
 
 {{< figure src="/media/docs/alerting/alerting-alertmanager-architecture.png" max-width="750px" alt="A diagram with the alert generator and alert manager architecture" >}}
 
-**Grafana Alertmanager**
+This architecture decouples alert rule evaluation from notification handling, allowing Grafana alerts to be forwarded to other alertmanagers.
 
-Grafana has its own built-in Alertmanager, referred to as "Grafana" in the user interface. It is the default Alertmanager and can only handle Grafana-managed alerts.
+Grafana can be configured to forward alerts to different types of alertmanagers. It’s important to note that each alertmanager manages its own independent alerting resources, such as:
 
-**Cloud Alertmanager**
+- Contact points and notification templates
+- Notification policies and mute timings
+- Silences
+- Active notifications
 
-Each Grafana Cloud instance comes preconfigured with an additional Alertmanager (`grafanacloud-STACK_NAME-ngalertmanager`) from the Mimir (Prometheus) instance running in the Grafana Cloud Stack. The Cloud Alertmanager can handle both Grafana-managed and data source-managed alerts.
+The `Choose Alertmanager` menu on these pages allows you to switch between alertmanagers.
 
-**Other Alertmanagers**
+{{< figure src="/media/docs/alerting/alerting-choose-alertmanager.png" max-width="750px" alt="A screenshot choosing an Alertmanager in the notification policies UI" >}}
+
+## Types of Alertmanagers in Grafana
+
+Grafana can be configured to forward both Grafana-managed and data source-managed alerts to various alertmanagers, depending on your infrastructure and alerting requirements.
+
+- **Grafana Alertmanager**: Grafana has its own built-in Alertmanager, referred to as "Grafana" in the user interface. It is the default Alertmanager and can only handle Grafana-managed alerts.
+
+- **Cloud Alertmanager**: Each Grafana Cloud instance comes preconfigured with an additional alertmanager (`grafanacloud-STACK_NAME-ngalertmanager`) from the Mimir (Prometheus) instance running in the Grafana Cloud Stack.
 
-Grafana Alerting also supports sending alerts to other alertmanagers, such as the [Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/), which can handle Grafana-managed alerts and data sources-managed alerts such as alerts from Loki, Mimir, and Prometheus.
+  The Cloud Alertmanager is available exclusively in Grafana Cloud and can handle both Grafana-managed and data source-managed alerts.
```

Comment:
```suggestion
  The Cloud Alertmanager is available exclusively in Grafana Cloud and can handle both Grafana-managed and data source-managed alerts.
```

Reply:
the alert term again - shall we specifically say alert rules and instances?

Reply:
we've using the term alerts in the context of notification handling. - I'd leave it as it is until we decide standarizing the term.